src/platforms/php/guides/symfony/index.mdx

```diff
@@ -17,32 +17,106 @@ Install the `sentry/sentry-symfony` package:
 composer require sentry/sentry-symfony
 ```
 
-### Configure
+<Note>
 
-Add your DSN to `app/config/config.yml`:
+Due to a bug in all versions below `6.0` of the [`SensioFrameworkExtraBundle`](https://github.com/sensiolabs/SensioFrameworkExtraBundle) bundle, you will likely receive an error during the execution of the command above related to the missing `Nyholm\Psr7\Factory\Psr17Factory` class. To workaround the issue, if you are not using the PSR-7 bridge, please change the configuration of that bundle as follows:
 
-```yaml {filename:app/config/config.yml}
+```yaml
+sensio_framework_extra:
+   psr_message:
+      enabled: false
+```
+
+For more details about the issue see [https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710](https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710).
+
+</Note>
+
+## Configure
+
+Add your DSN to `config/packages/sentry.yaml`:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    dsn: "___PUBLIC_DSN___"
+```
+
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
+
+```php {filename:config/bundles.php}
+<?php
+
+return [
+    // ...
+    Sentry\SentryBundle\SentryBundle::class => ['all' => true],
+];
+```
+
+### Monolog Integration
+
+If you are using [Monolog](https://github.com/Seldaek/monolog), you also need these additional configuration and services to log the errors correctly:
+
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
-  dsn: "___PUBLIC_DSN___"
+    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
+
+monolog:
+    handlers:
+        sentry:
+            type: service
+            id: Sentry\Monolog\Handler
+
+services:
+    Sentry\Monolog\Handler:
+        arguments:
+            $hub: '@Sentry\State\HubInterface'
+            $level: !php/const Monolog\Logger::ERROR
+```
+
+Additionally, you can register the `PsrLogMessageProcessor` to resolve PSR-3 placeholders in reported messages:
+
+```yaml {filename:config/packages/sentry.yaml}
+services:
+    Monolog\Processor\PsrLogMessageProcessor:
+        tags: { name: monolog.processor, handler: sentry }
 ```
 
-If you **are not** using Symfony Flex, you'll also need to enable the bundle in `app/AppKernel.php`:
+## Test the implementation
 
-```php {filename:app/AppKernel.php}
+To test that both logger error and exception are correctly sent to sentry.io, you can create the following controller:
+
+```php
 <?php
-class AppKernel extends Kernel
+
+namespace App\Controller;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SentryTestController
 {
-    public function registerBundles()
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
```

Comment:
Do we not want to promote best practise?
```suggestion
    private LoggerInterface  $logger;
```

Reply:
This depends on the PHP Version that it works and so I would avoid this. As the code should also work with older versions I would not use PHP 7.4 specific code if not really necessary to test the sentry implementation.